ui/packages/admin-core-logviewer/src/main/webapp/js/filter.js

```diff
@@ -40,7 +40,9 @@ export default (filters, logs) => {
   const regexps = fields.reduce((o, field) => {
     try {
       o[field] = new RegExp(filters[field], 'i')
-    } catch (e) {}
+    } catch (e) {
```

Comment:
❓ Why not just get rid of the `try/catch` completely instead?

Reply:
I don't want the exception to bubble up